pkg/clients/rds/rds.go

```diff
@@ -170,12 +169,6 @@ func (c *client) DeleteDBInstance(id string) error {
 	return err
 }
 
-// LateInitialize fills the empty fields in *v1alpha1.RDSInstanceParameters with
-// the values seen in rds.DBInstance.
-func LateInitialize(in *v1alpha1.RDSInstanceParameters, db *DBInstance) {
-	in.Engine = db.Engine
-}
```

Comment:
Was this previously unused? Do we need to be late initializing?

Reply:
Needed and reverted the removal. Thanks for the reminder.